pulsar-common/src/main/java/org/apache/pulsar/common/util/netty/EventLoopUtil.java

```diff
@@ -122,6 +122,16 @@ public static Class<? extends ServerSocketChannel> getServerSocketChannelClass(E
         }
     }
 
+    public static Class<? extends SocketChannel> getSocketChannelClass(EventLoopGroup eventLoopGroup) {
+        if (eventLoopGroup instanceof IOUringEventLoopGroup) {
+            return IOUringSocketChannel.class;
+        } else if (eventLoopGroup instanceof EpollEventLoopGroup) {
+            return EpollSocketChannel.class;
+        } else {
+            return NioSocketChannel.class;
+        }
+    }
+
```

Comment:
```suggestion
```

Duplication code, please see `getClientSocketChannelClass`.

Reply:
thanks, i missed it